hack/generator/pkg/codegen/code_generator.go

```diff
@@ -165,17 +171,118 @@ func loadSchema(source string) (*gojsonschema.Schema, error) {
 	return schema, nil
 }
 
-//TODO: Only clean generated files
-func cleanFolder(outputFolder string) error {
-	err := os.RemoveAll(outputFolder)
+func deleteGeneratedCodeFromFolder(outputFolder string) error {
+	globPattern := path.Join(outputFolder, "**", "*", "*"+astmodel.CodeGeneratedFileSuffix) + "*"
```

Comment:
Should this finish with `.go`?

Reply:
Yeah, probably. I'll move it into the constant so that we don't repeat ourselves all over too.